jaggery-modules/carbon-metrics/feature/etc/feature.properties

```diff
@@ -0,0 +1,242 @@
+################################################################################
+# Copyright 2009 WSO2, Inc. (http://wso2.com)
```

Comment:
Thanks @thushara35 , I have fixed it in [this commit](https://github.com/wso2/carbon-store/pull/844/commits/8a9d95fd1f9bd681e20b44f2ab4f05d3833fbd62)